api/PluginArray.json

```diff
@@ -20,7 +20,7 @@
             "version_added": "4"
           },
           "ie": {
-            "version_added": null
+            "version_added": "11"
```

Comment:
This should match https://mdn-bcd-collector.appspot.com/tests/api/Navigator/plugins but that tests true in IE10. Can you test an instance (`navigator.plugins`) here?